recipe/run_test.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 set -xeuo pipefail
 
-CORRECTNESS="${RECIPE_DIR}/python_bindings/correctness"
```

Comment:
correctness tests rely on GenGen.cpp and other files that aren't included in the halide install. Should the be?

Reply:
GenGen.cpp *should* be included in the Halide install.